### ISSUE TITLE
feature: asking k8s-bot to rerun all failed tests

### ISSUE THREAD

**Issue by jessfraz** [kubernetes/contrib#2167]
Something like "k8s-bot rerun failed tests" so that you dont have to copy paste all the commands for each test individually or rerun all.

**Comment by jessfraz** [comment]
Just seeing if other people would find this useful

**Comment by eparis** [comment]
yes, but it needs to be in the test-infra repo, since that's where this would be implemented.
